pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
```

Comment:
Changes in this file and `poetry.lock` were copied from the [base branch](https://github.com/airtop-ai/airtop-python-sdk/pull/15) to resolve dependency-related compilation errors and verify the rest of the compilation succeeded